### PR TITLE
Rwanda CHOGM 2022

### PR DESCRIPTION
**Pull request by mbonabucya**
## Rwanda CHOGM 2022

add linters configuration
Add index.html ,  index.js , style.css
add about.html ,about.css , about.js
add Pop up menu (hamburger button ) to all pages
[Liveo Demo Video](https://www.loom.com/share/e1d5d0c1844a402eb3c4fb5deed9ae78)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,99 @@
...
+# Microverse-capstone-1---CHOGM Rwanda 2022
+![](https://img.shields.io/badge/CapstoneProject-orange)
+
+## Description
+
+The 2022 Commonwealth Heads Of Government Meeting, Also Known As CHOGM 2022, Will Be The 26th Meeting Of The Heads Of Government Of The Commonwealth Of Nations.
+The Meeting Was Originally Scheduled For 26–27 June 2020 Preceded By Various Fora Between 22–25 June, But Was Postponed Twice Due To The COVID-19 Pandemic.
+
+> Learning objectives: 
+
```
Comment (KDlamini) [inline README.md:10]: Kindly record a video presenting the features of the project with a maximum length of 5 minutes which must contain a clear and objective description about the project.

- You can either attach the link on the README or PR description.

File: README.md [hunk 2]
```diff
@@ -1 +1,99 @@
...
+ - GitHub: [@mbonabucya](https://github.com/mbonabucya)
+- Twitter: [@Mbonabucya12](https://twitter.com/Mbonabucya12)
+- LinkedIn: [@James-mbonabucya](https://linkedin.com/in/james-mbonabucya)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```
Comment (KDlamini) [inline README.md]: Kindly add a working link to the `issues` page in the README.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+      <a href="index.html" class="logo"
+        ><img src="images/logo.png" alt="chogm-logo"
+      /></a>
+      <nav class="navbar">
+        <a href="about.html" class="nav-link">about</a>
+        <a href="#program" class="nav-link">program</a>
+        <a href="#join" class="nav-link">join</a>
+        <a href="#sponsor" class="nav-link">sponsor</a>
+        <a href="#news" class="nav-link">news</a>
+      </nav>
```
Comment (KDlamini) [inline index.html]: The desktop size navbar must be divided into a top dark bar and a bottom white bar. The top short bar section with social media links, logout, and language links aligned to the right as per [Behance design ](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)specifications.

Your design:
image

Behance design:
image

File: about.html [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+          <div class="col-sm-6">
+            <img src="images/logo.png" alt="logo-footer" />
+          </div>
+          <div class="col-sm-6 ">
+            <div class="row footer-p">
+              <p>CHOGM Rwanda 2022 Some rights are reserved<br/>
+                달리 정하지 않은 웹 사이트 의 모든 내용 은 크리에이티브 커먼즈 저작자 표시 2.0
+                대한민국 라이선스 에 따라 이용할 수 있습니다 . 자세한 이용 기능 내용 과 허락 조건 은 http://eduport.org/licences/by/2.0/kr 를 보세요 .
+              </p>
+            </div>
```
Comment (KDlamini) [inline about.html]: The desktop size About Page footer should have have a dark background.

Your about footer:
image

Behance design About footer:
image

File: about.html [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    </section>
+    <section class="logo-design">
+      <h2>Commonwealth Heads of Government Meeting 2022 (CHOGM)</h2>
+      <img src="images/logo.png" alt="logo-description" />
+    </section>
+    <section class="history">
+      <h3>see the past CHOGM Meetings</h3>
+      <h4>
+        take look at the last two CHOGM meetings that held at Geneva and UK
+      </h4>
```
Comment (KDlamini) [inline about.html]: Kindly make sure that the past meetings cards have a visible background image with a background color overlay, year, and event description.

Your past event cards:
image

Behance past events:
image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @mbonabucya 🙋‍♀️,  

Excellent work! 💯 
You have done a great job creating a conference page using the design specifications.🥳 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  The [JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) & [HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) linters are set up correctly
✔️  Appropriate credit to [Cindy Shin](https://www.behance.net/adagio07)appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
✔️  The pages use the same colors, for text, backgrounds, borders, etc, from the original design.
✔️  Correct PR description
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @mbonabucya, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title :heavy_check_mark:
 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**